community/modules/file-system/nfs-server/variables.tf

```diff
@@ -47,10 +47,43 @@ variable "type" {
   default     = "pd-ssd"
 }
 
+# Deprecated, replaced by instance_image
+# tflint-ignore: terraform_unused_declarations
 variable "image" {
   description = "the VM image used by the nfs server"
   type        = string
-  default     = "cloud-hpc-image-public/hpc-centos-7"
+  default     = null
+
+  validation {
+    condition     = var.image == null
+    error_message = "The 'image' setting is deprecated, please use 'instance-image' with the sub-settings 'project' and 'family' or 'name'."
```

Comment:
Make same text changes as advised above. Make sure `git grep instance-image` doesn't find any hits and same for "sub-settings"

Reply:
Done